chainer/_sphinx_ext.py

```diff
@@ -0,0 +1,40 @@
+import sphinx
```

Comment:
I think it's better to place this file under `docs/source` with name like `_napoleon_patch.py` or sth.
Then `extensions = ['sphinx.ext.autodoc', ... , '_napoleon_patch']` in `conf.py` should work.